### DISCUSSION TITLE
Use X colors

### DISCUSSION THREAD

**Discussion by charlie45000** [davatorium/rofi#1465] [Ideas/Feature Requests]
Hello!
Since the possibility to directly use Xresources has been removed, would it be possible to be able to use at least the 16 colors model (color0 to color15)? 
An other idea would be to allow to use Xresource property in the config file (as in the i3wm config file for exemple. Something like: `from_resource(rofi.foreground, default_value);`), I don't know if it would keep the problems that lead to the end of this configuration mode solved. Xresources was a convenient way to have the same properties on many programs (mostly for colors (including background and foreground) and fonts) and if it was possible to get only that back (and not configuring it entirely like that if it was too hard to maintain), it would be great!

Have a nice day!

**Comment by DaveDavenport** [comment]
I did suggest that in the past to have as an option.. 
Two main reasons why it was not implemented:
* nobody at the time indicated interest in it.
* it would create an incompatibility in configuration file with a wayland port.
